middleware/keyauth/keyauth_test.go

```diff
@@ -0,0 +1,461 @@
+//nolint:bodyclose // Much easier to just ignore memory leaks in tests
+package keyauth
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/utils"
```

Comment:
Should this use these utils or the utils repo?

Reply:
We will use utils repo in v3